source/bigint.c

```diff
@@ -559,3 +559,79 @@ int aws_bigint_subtract(struct aws_bigint *output, struct aws_bigint *lhs, struc
 
     return AWS_OP_SUCCESS;
 }
+
+int aws_bigint_multiply(struct aws_bigint *output, struct aws_bigint *lhs, struct aws_bigint *rhs) {
+    struct aws_bigint temp_output;
+    AWS_ZERO_STRUCT(temp_output);
```

Comment:
optional optimization idea:
If `output` is NOT an alias of `lhs` or `rhs`, then you don't need to use `temp_output`